scaleapi/__init__.py

```diff
@@ -273,6 +273,9 @@ def tasks(self, **kwargs) -> Tasklist:
                 If true, returns a pre-signed s3 url for the
                 attachment used to create the task.
 
+            limited_response (bool):
+                If true, returns limited task response.
```

Comment:
can we document what fields are returned in the limited_response?

Reply:
Agree, we can both document it in this comment section as well as document it in the Readme with more details.

Reply:
Got it! Just updated with the new version specifying fields to be returned as comments and also added to the readme